frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx

```diff
@@ -44,80 +57,122 @@ const PredictedDataPage = function () {
-    //         data: new Uint8Array([5, 6]),
-    //         predictionType: null,
-    //         dateTime: new Date(),
-    //     },
-    // ];
-
-    // const appleWatchFiles = [
-    //     {
-    //         id: 123,
-    //         data: new Uint8Array([1, 2]),
-    //         predictionType: null,
-    //         dateTime: new Date(),
-    //     },
-    //     {
-    //         id: 456,
-    //         data: new Uint8Array([5, 6]),
-    //         predictionType: null,
-    //         dateTime: new Date(),
-    //     },
-    // ];
-
-    const appleWatchPredictedFiles: Array<PredictedFileWithType> =
-        appleWatchFiles?.length !== 0
-            ? appleWatchFiles.map((file: FileData) => ({
-                  ...file,
-                  watch: WatchType.APPLE_WATCH,
-                  name: `${WatchType.APPLE_WATCH} ${file.id.toString()}`,
-                  isSelected: false,
-              }))
-            : [];
-
-    const fitbitPredictedFiles: Array<PredictedFileWithType> =
-        fitBitFiles?.length !== 0
-            ? fitBitFiles.map((file: FileData) => ({
-                  ...file,
-                  watch: WatchType.FITBIT,
-                  name: `${WatchType.FITBIT} ${file.id.toString()}`,
-                  isSelected: false,
-              }))
-            : [];
-
-    const availableFiles: Array<PredictedFileWithType> =
-        fitbitPredictedFiles.concat(appleWatchPredictedFiles);
+        const appleWatchFiles = [
+            {
+                id: 123,
+                data: new Uint8Array([1, 2]),
+                predictionType: null,
+                dateTime: new Date(),
+            },
+            {
+                id: 456,
+                data: new Uint8Array([5, 6]),
+                predictionType: null,
+                dateTime: new Date(),
+            },
+        ];
+        // #endregion
+
+        const appleWatchPredictedFiles: Array<PredictedFileWithType> =
+            appleWatchFiles?.length !== 0
+                ? appleWatchFiles.map((file: FileData) => ({
+                      ...file,
+                      watch: WatchType.APPLE_WATCH,
+                      name: `${WatchType.APPLE_WATCH} ${file.id.toString()}`,
+                      isSelected: false,
+                  }))
+                : [];
+
+        const fitbitPredictedFiles: Array<PredictedFileWithType> =
+            fitBitFiles?.length !== 0
+                ? fitBitFiles.map((file: FileData) => ({
+                      ...file,
+                      watch: WatchType.FITBIT,
+                      name: `${WatchType.FITBIT} ${file.id.toString()}`,
+                      isSelected: false,
+                  }))
+                : [];
+
+        return appleWatchPredictedFiles.concat(fitbitPredictedFiles);
+    };
+
+    setAvailableFiles(getAvailableFiles());
 
     /**
-     * mapping avaliable files to html components
+     * Request available files list from API
+     */
+    const refreshAvailableFiles = () => {
```

Comment:
i know we have this in the old designs, but im wondering if we still need this. Since everytime we load or click into the page it automatically refreshes because of how our api hooks are designed.

Reply:
is what you're saying then that we do not need refresh button? it's not that much rerendering to just refresh the page and I think that this is how the user would expect to refresh the file list anyways

Reply:
I think Ralph is saying that the pages are re-rendering automatically anyways (react states) and the hooks run on page re-renders, so we're already fetching the data anyways. 

I believe the hook for getting predicted data is in a useEffect so it should run on state variable changes